### PR TITLE
Forgot Health Container 

### PR DESCRIPTION
**Pull request by Petrica81**
Forgot to commit the changes from Scenes, which prevents an error from occurring.

### CODE REVIEW

**Review by Denj1X — APPROVED** [review]
So our character has 3 pills in order to survive. The user has a slightly better chance of winning.

File: Assets/Scenes/SampleScene.unity [hunk 1]
```diff
@@ -205,11 +205,18 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 5ea89fc4b9a29fe469c090277c774218, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  piles: []
```
Comment (widdrr) [inline Assets/Scenes/SampleScene.unity:208]: Shouldn't it be pills/pill? instead of pile/piles?

**Review by widdrr — CHANGES_REQUESTED** [review]

**Review by RobertKeanu — APPROVED** [review]
lgtm

File: Assets/Scenes/SampleScene.unity (hunk 1 above)
Comment (Petrica81) [reply Assets/Scenes/SampleScene.unity:208]: Fixed

File: Assets/Scripts/HealthManager.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 public class HealthManager : MonoBehaviour
 {
-    public Image[] piles;
+    public Image[] pills;
     public Sprite fullPile;
```
Comment (widdrr) [inline Assets/Scripts/HealthManager.cs:9]: you missed some

**Review by widdrr — CHANGES_REQUESTED** [review]

File: Assets/Scripts/HealthManager.cs (hunk 2 above)
Comment (Petrica81) [reply Assets/Scripts/HealthManager.cs]: Fixed.

**Review by widdrr — APPROVED** [review]
lgtm

**Review by tiberiusss26 — APPROVED** [review]
lgtm

**Pull request merged**